tests/Helper/files/cli-config.lala

```diff
@@ -0,0 +1 @@
+something wrong
```

Comment:
Please make sure your IDE is set to *Ensure line feed at file end on Save* (ex. JetBrains, or similar!)